articles/native-platforms/ios-swift/08-calling-apis.md

```diff
@@ -0,0 +1,75 @@
+---
+title: Calling APIs
+description: This tutorial will show you how to manage tokens to make authenticated API calls, using NSURLSession.
+---
+
+The reason for implementing authentication, in the first place, is to protect information. In this case, your information is a resource served from a server of any sort. Auth0 provides a squad of tools to assist you with end-to-end authentication in an application. We recommend that you conform to RFC standards by sending valid authentication tokens through an authorization header.
```

Comment:
@pablolvillar The doc is missing the system requirements and sample download link.